ai-data/llm-inference/how-to/monitor-deployment.mdx

```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: How to monitor an LLM Inference deployment
+  description: This page explains how to monitor an LLM Inference deployment
+content:
+  h1: How to monitor an LLM Inference deployment
+  paragraph: This page explains how to monitor an LLM Inference deployment
+tags: ai monitoring
+dates:
+  validation: 2024-03-06
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+This documentation page shows you how to monitor your LLM Inference deployment with [Cockpit](/observability/cockpit/quickstart/).
+
+<Macro id="requirements" />
+
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+  - An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

Comment:
Cockpit activation is going to be deprecated on the 2nd of April. Cockpits will be automatically activated. Should we remove this requirement?